### ISSUE TITLE
Choice of license

### ISSUE THREAD

**Issue by Jacq4nn** [UBC-MDS/textfeatureinfo#56]
Examine the license for your project and consider whether this is the choice you want to make, or whether you want to change the license. Discuss and reason the license choice by opening issues in both Python and R repositories. As it is likely to be a very similar discussion for both projects, one of these issues can just link to the other issue where it is thoroughly dicussed.

**Comment by Jacq4nn** [comment]
We are currently using the MIT License in both our R and Python package. 

The aim of this package is to allow data scientists to have text feature engineering tools that are simple and quick to employ. Moreover, we have only just started coming up with these tools and there are so many more functions that we have yet to explore. As such, I feel that it would be in our interest to allow the data science community the creative freedom to use our functions, be involved in our work, and allow them to make modifications to our functions. Moreover, having an easily recognisable license will not act as a deterrent for future contributors or collaborators. 

With these considerations in mind, I feel that the MIT License would be more suited, compared to CNU GPLv3 (downside: need to state change) for example.

**Comment by lynnwbl** [comment]
I agree with your comment. A MIT license would be appropriate for our R and Python package, since we want a license that protects our project and is simple and permissive. There is still space for improvement in our packages and we welcome people to freely use and collaborately modify our packages. Although both MIT and GNU licences allow modifications and distributions, MIT license is simple and permitting distributing closed source versions.

**Comment by paradise1260** [comment]
I am also happy with MIT license for both R and Python package. It is simple, and most developers are familiar with it. Also, I would be happy if anyone can build upon our functions, make improvements or add new functionalities to our package. Comparing MIT and GNU, I prefer MIT for its simplicity and few number of conditions. The conditions that GNU license has might prevent people from collaborating with us.

**Comment by kphaterp** [comment]
After visiting the ['choose a license'](https://choosealicense.com/) website, I concur that the MIT license is the most suitable license for both our Python project and our R project. It is clear that the R and Python packages that we have created are unfinished and further improvements to the package should be invited and encouraged. By using the MIT license, we retain protective rights as the authors of the package, while also encouraging contributions from the community. Not only this, but our package is quite reproducible and merely created as a means to simplify an otherwise tedious task. I don't believe that our code is original or groundbreaking enough to justify the use of a licence other than MIT.
